### ISSUE TITLE
No popup for incoming calls

### ISSUE THREAD

**Issue by jurgenhaas** [sagetarian/skype-wrapper#79]
Using latest skype-wrapper on Ubuntu 12.10 with Skype 4.1 and it works really great. My only problem is that incoming calls are not indicated by any popups, I just hear it ringing.

After some time I found out that I have to do a right-click on the top-most contact in the conversation list and there is an item that allows me to accept the incoming call.

Is that a bug, a missing feature or some mis-configuration at my end?

**Comment by grissir** [comment]
Popup are managed by Skype so check your Skype settigns but I know that sometimes the Skype client fails to render the pop up right

**Comment by jurgenhaas** [comment]
Can't find any popup settings in the Skype client. There are notification settings but I can only define sound alerts, nothing about a popup or the likes.

**Comment by grissir** [comment]
As I believe you are german:
Ja wie gesagt pop ups stammen vom Skype client selber und werden nicht von Skype wrapper beeinflusst aber auf meinen PC werden sie au nicht zuverlässig angezeigt. Ich verwende aber auch 13.04 und ging bisher davon aus, dass es daran lag dass ubuntu 13.04 nicht unterstützt ist.
Meiner Meinung nach ist es ein bug im Skype client .
Btw ich werde bald damit beginnen einen eigenen client zu schreiben auf Basis von Skype kit ( dieser kann dann nicht open source sein)

**Comment by jurgenhaas** [comment]
Ja, ich denke auch es ist wohl ein Fehler im Skype-Client. Wen kann man denn da kontaktieren wenn man nicht von der offiziellen Skype-Seite sondern das Paket mit apt installiert hat?

Das mit einem neuen Client hört sich nicht schlecht an. Wenn Du Hilfe brauchst? Konzept, Ideen, Beta-Tests, ... alles schon gemacht und ich gebe gerne Input.

**Comment by grissir** [comment]
So weit ich weiß einfach auf der Skype Seite weil canonical einfach nur den offiziellen client in ihre repositorys kopiert.
Ich melde mich wenn ich Hilfe brauche :) wird ne qml APP werden (ubuntu touch)

**Comment by jurgenhaas** [comment]
(Back to english so that everyone can benefit from the information)

I have done some more tests and want to report back. It is not resolved yet but it is certainly a Skype issue and not an issue with skype-wrapper. Here you go:

First, I have updated to Ubuntu 13.04 with no improvement regarding this issue. My new issue now is that skype-wrapper doesn't launch anymore, but that is off topic here.

I then contacted Skype support (live chat) and we went through the problem and they couldn't help and forwarded the issue to the back office. They contacted me but couldn't help yet.

Then by accident, I realized that when I'm working with the notebook alone (without an external monitor), then the notification is displayed properly. So IT IS WORKING correctly. As soon as I plugin an external monitor which extends my desktop to the right, the notification is missing. To double check I unplugged the external monitor again and the notification was there again.

What I haven't tried yet if the resolution of my monitors has an effect (Laptop 1600_900, External 1366_768). I could imagine that with the given resolutions the notification is in the bottom right corner of the external monitor but at 900 pixels vertically and that would be outside of the viewport.

Could everyone else with the same problem try and do the same test in their environment to confirm this analysis?

**Comment by grissir** [comment]
Hi
Is Skype or skype-wrapper not working? Skype isn't launching with nvidia and and graphic chips on Ubuntu 13.04 for a workaround search webupd8.org .
Please also note that you have to reboot your PC to ensure that skype-wrapper is correctly working. The latest version is intended to use the standard Skype launcher icon and mustn't be used by an other way than launching Skype normal

**Comment by jurgenhaas** [comment]
Yes, I do have problems starting it. Neither directly nor indirectly is it launching. When I try it in the terminal I get the following error:
`/usr/lib/python2.7/dist-packages/gobject/constants.py:24: Warning: g_boxed_type_register_static: assertion `g_type_from_name (name) == 0' failed`

I've tried everything: reinstalling, rebooting, etc. nothing helps. And yes, I do have NVidia graphic chips so that may explain it. What exactly should I look for when I search webupd8.org? Do they have a workaround in one of their blogs? Haven't seen anything at a first glance.

**Comment by grissir** [comment]
http://m.webupd8.org/2013/04/fix-skype-not-working-in-ubuntu-1304.html

**Comment by jurgenhaas** [comment]
Hmm, that link describes on how to get a Skype problem fixed, but Skype is running just fine on my Ubuntu 13.04 even without that fix. I only have the problem with skype-wrapper, not with Skype

**Comment by grissir** [comment]
Sorry thought it was a bug in Skype I will have a look on this but for me everything is working fine
